### ISSUE TITLE
Handle Win+... hotkeys better

### ISSUE THREAD

**Issue by zulc22** [cairoshell/cairoshell#743]
**Is your feature request related to a problem? Please describe.**
I very consistently use Win key hotkeys and when Cairo is set to my default shell, the Programs window gets in the way and not only distracts me but typically also prevents my hotkeys from working. Try pressing Win+`→` or Win+`.` with Cairo set to the default shell.

**Describe the solution you'd like**
Do not open the 'Programs' menu until the Win key is released with no other keyboard keys pressed.

**Describe alternatives you've considered**
A way to disable or customize the Cairo 'Programs' hotkey would be a usable workaround, but is probably worth adding since there's a customizable Cairo menu hotkey anyway.

**Comment by zulc22** [comment]
i genuinely do not want to have to use cairo on top of explorer because cairo genuinely seems like it's a lot lighter on my system

**Comment by iki** [comment]
Blocker for me too

I'd love to use Cairo, but Win key is at center of many shortcuts I use (both the default ones and custom ones using AutoHotKey) and they don't work with Cairo running

**Comment by librix** [comment]
WIn+num is my shortcuts key focus, and locations and muilt-rows of icons in 'task bar' is more important for me.

**Comment by dremin** [comment]
Duplicate of #713
